ert3/evaluator/_evaluator.py

```diff
@@ -1,57 +1,136 @@
 import pickle
-from typing import Dict
+import time
+from typing import Dict, Optional, Union, Any
+from concurrent import futures
+import sys
 
 import ert
 from ert_shared.ensemble_evaluator.config import EvaluatorServerConfig
+from ert_shared.models.base_run_model import BaseRunModel
+from ert_shared.status.tracker.evaluator import EvaluatorTracker
+from ert_shared.cli.monitor import Monitor
 from ert_shared.ensemble_evaluator.ensemble.base import Ensemble
-from ert_shared.ensemble_evaluator.entity.identifiers import EVTYPE_EE_TERMINATED
+from ert_shared.ensemble_evaluator.entity.identifiers import (
+    EVTYPE_EE_TERMINATED,
+)
 from ert_shared.ensemble_evaluator.evaluator import EnsembleEvaluator
 
 from ert_shared.status.entity.state import (
     ENSEMBLE_STATE_STOPPED,
     ENSEMBLE_STATE_FAILED,
-    REALIZATION_STATE_FINISHED,
 )
 
 
+class DummyBaseRunModel(BaseRunModel):
+    def __init__(self, phase_count: int = 1):
+        self._phase: int = 0
+        self._phase_count: int = phase_count
+        self._phase_name: str = "Starting..."
+        self._job_start_time: int = 0
+        self._job_stop_time: int = 0
+        self._fail_message: str = ""
+        self._failed: bool = False
+
+    def create_context(self, arguments: Any) -> Any:
+        return None
+
+    def runSimulations(self, arguments: Any) -> Any:
+        return self.create_context(arguments)
```

Comment:
We start simulations wholly differently in ERT3. Is this needed?

Reply:
yes, as long as we still use `BaseRunModel`, some functions need to be implemented; eg. `runSimulations` and `teardown_context`.

Reply:
But we shouldn't use `BaseRunModel`.

Reply:
Now it's a standalone `ERT3RunModel` class.